README.md

```diff
@@ -451,6 +451,28 @@ Add the load module to the status modules list.
 set -g @catppuccin_status_modules_right "... load ..."
 ```
 
+### Git module
+
+#### Requirements
+This module depends on [gitmux](https://github.com/arl/gitmux).
+
+#### Install
+The preferred way to install gitmux is using [Binary release](https://github.com/arl/gitmux?tab=readme-ov-file#installing).
+[Download the latest](https://github.com/arl/gitmux/releases/latest) binary for your platform/architecture and uncompress it.
+
+#### Configure
+Add the git module to the status modules list.
+```sh
+set -g @catppuccin_status_modules_right "... git ..."
+```
+
+#### Customizing the git module
+To customize the git module, you can follow the instruccitons in the [gitmux documentation](https://github.com/arl/gitmux/blob/main/README.md#customizing). Then uncomment the following line in your `git.sh` file:
+```
+  text="$(  get_tmux_option "@catppuccin_git_text"  "#(gitmux -cfg $HOME/.gitmux.conf "#{pane_current_path}")")"
+```
```

Comment:
Move this under `#### Configure` as `set -g @catppuccin_git_text "#(gitmux -cfg $HOME/.gitmux.conf \"#{pane_current_path}\")")" # [custom config](https://github.com/arl/gitmux/blob/main/README.md#customizing).` or so

Reply:
Changed this to:

#### Configure
Add the gitmux module to the status modules list.
```sh
set -g @catppuccin_status_modules_right "... gitmux ..."
```

To customize the gitmux module, you can follow the instruccitons in the [gitmux documentation](https://github.com/arl/gitmux/blob/main/README.md#customizing) and add this line in your tmux configuration:
```sh
set -g @catppuccin_git_text "#(gitmux -cfg $HOME/.gitmux.conf \"#{pane_current_path}\")"
```

Let me know if this is correct